src/applications/vaos/appointment-list/components/VistaSchedulingServiceAlert.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import InfoAlert from '../../components/InfoAlert';
+import { selectFeatureVaosV2Next } from '../../redux/selectors';
+import { selectBackendServiceFailuresInfo } from '../redux/selectors';
+import { FETCH_STATUS } from '../../utils/constants';
+
+export default function VistaSchedulingServiceAlert() {
+  const featureVaosV2Next = useSelector(state =>
+    selectFeatureVaosV2Next(state),
+  );
+  const { backendServiceFailures, futureStatus } = useSelector(state =>
+    selectBackendServiceFailuresInfo(state),
+  );
+
+  if (futureStatus === FETCH_STATUS.succeeded) {
+    // const hasVistaServiceFailure = !!backendServiceFailures?.meta.find(
+    //   backendServiceFailure =>
+    //     backendServiceFailure?.system === 'VSP' &&
+    //     backendServiceFailure?.code === 10000,
+    // );
+    const hasVistaServiceFailure = true;
```

Comment:
@ryanshaw should this be set to true here?

Reply:
@simiadebowale I set it that way temporarily.